src/tours/documents.py

```diff
@@ -0,0 +1,46 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+from .models import Tours
+from categories.models import Categories
+from users.models import CustomUser
+from files.models import File
+
+
+@registry.register_document
+class TourDocument(Document):
+    created_by = fields.ObjectField(properties={
+        'email': fields.TextField(),
+        'phone': fields.TextField(),
+    })
+
+    images = fields.ObjectField(properties={
+        'name': fields.TextField(),
+        'link': fields.TextField(),
+    })
+ 
+
+    class Index:
+        name = 'tours'
+        settings = {'number_of_shards': 2,
```

Comment:
`number_of_shards` là để làm gì?

Reply:
ElasticSearch uses concept called Shard to distribute index documents across all nodes.
An index can potentially store a large amount of data that can exceed the hardware limits of a single node

Reply:
khi mình tạo 1 index thì mình phải định nghĩa có bao nhiêu shard , nếu ko định nghĩa thì nó mặc định là 5 à, nghĩa là có 5 shards chứa data của mình